links-slackbot/src/main/java/juja/microservices/links/slackbot/controller/LinksSlackbotController.java

```diff
@@ -1,25 +1,111 @@
 package juja.microservices.links.slackbot.controller;
 
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import juja.microservices.links.slackbot.exceptions.ExceptionsHandler;
+import juja.microservices.links.slackbot.model.Link;
+import juja.microservices.links.slackbot.service.LinksSlackbotService;
+import lombok.extern.slf4j.Slf4j;
+import me.ramswaroop.jbot.core.slack.models.RichMessage;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
 
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
+import java.io.PrintWriter;
+import java.net.HttpURLConnection;
+import java.util.Arrays;
+import java.util.stream.Collectors;
 
+/**
+ * @author Ivan Shapovalov
+ */
 @RestController
+@Slf4j
 @RequestMapping(value = "/v1/commands")
 public class LinksSlackbotController {
 
-    @PostMapping(value = "/links/add", consumes = MediaType.APPLICATION_FORM_URLENCODED_VALUE)
-    public void onReceiveSlashCommandActivateTeam(@RequestParam("token") String token,
-                                                  @RequestParam("text") String text,
-                                                  @RequestParam("response_url") String responseUrl,
-                                                  HttpServletResponse response) throws IOException {
-        //TODO implement feature SLB-F1
+    private final RestTemplate restTemplate;
+    private final LinksSlackbotService linksSlackbotService;
+    private final ExceptionsHandler exceptionsHandler;
 
+    @Value("${slack.slashCommandToken}")
+    private String slackToken;
+    @Value("${message.sorry}")
+    private String SORRY_MESSAGE;
```

Comment:
Is your private String SORRY_MESSAGE a constant?

Reply:
Of course not. Fixed